### DISCUSSION TITLE
Implement error recovery for radio buttons: clicking on a selected radio button should unselect it

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#186] [General Survey Design]
One of @michaelquiapos’s points was that radio buttons do not provide any way for error recovery out of the box: once you’ve selected a radio button, you can only select a different one in the group, but you cannot clear the selection entirely.

An unobtrusive way to facilitate error recovery without cluttering the UI is to implement that if a user clicks on a selected radio button, that unselects it (like a checkbox).

![radios](https://github.com/Devographics/surveys/assets/175836/e68d4c0f-02ad-41f4-a951-0e50649ec804)

This could likely be implemented in a generic way, for all radio buttons.

**Comment by SachaG** [comment]
Good idea!

**Comment by SachaG** [comment]
Done.
